anti-analysis/anti-debugging/debugger-detection/check-for-debugger-via-api.yml

```diff
@@ -18,3 +19,4 @@ rule:
       - api: WUDFPlatform.WudfIsAnyDebuggerPresent
       - api: WUDFPlatform.WudfIsKernelDebuggerPresent
       - api: WUDFPlatform.WudfIsUserDebuggerPresent
+      - property: System.Diagnostics.Debugger::IsAttached
```

Comment:
add `matches: compiled to the .NET platform` to set a good example? (see https://github.com/mandiant/capa/discussions/1106#discussioncomment-3202916)

Reply:
this rule is pretty clear but we should come up with a consistent solution

Reply:
I like the idea of `matches: xyz technology` where we expect the former to be included in all rule blocks matching `xyz` technology. We can then maintain the one`xyz technology` rule.